sonar-plugin-api/src/main/java/org/sonar/api/server/rule/RulesDefinition.java

```diff
@@ -811,16 +811,24 @@ public NewRule setDebtRemediationFunction(@Nullable DebtRemediationFunction fn)
     }
 
     /**
+     * @deprecated since 5.5, replaced by {@link #setGapDescription(String)}
+     */
+    @Deprecated
+    public NewRule setEffortToFixDescription(@Nullable String s) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)